### DISCUSSION TITLE
Add ability to download offline blocks editor without a real robot

### DISCUSSION THREAD

**Discussion by qrAIhttps** [FIRST-Tech-Challenge/FtcRobotController#1193] [Ideas] [answered]
This will be helpful for programmers wanting to start code before the robot is built.

**Comment by texasdiaz** [comment, marked as answer]
> This will be helpful for programmers wanting to start code before the robot is built.

Unfortunately robot configs are very hardware dependent (you cannot currently "simulate having an expansion hub or a Webcam" presently). However, you can gather your electronics, connect it together quickly, and create a benchtop config on your control hub. Then you can export the offline blocks editor, and give the equipment back to the build team. That's as good as it's going to get in afraid.
